templates/web/php-fpm.d.zabbix.conf.epp

```diff
@@ -0,0 +1,26 @@
+[zabbix]
```

Comment:
please add a header like `# THIS FILE IS MANAGED BY PUPPET`

Reply:
Will do.